.github/workflows/pre-commit.yml

```diff
@@ -8,18 +8,13 @@ jobs:
   pre-commit:
     name: Detecting code style issues
     runs-on: ubuntu-latest
+    container: holzhaus/mixxx-ci:20210526
```

Comment:
We could then also add a comment here where the corresponding Dockerfile for generating this image could be found in the repo.